naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftStore.java

```diff
@@ -155,13 +157,15 @@ private synchronized Datum readDatum(File file, String namespaceId) throws IOExc
             if (StringUtils.isBlank(json)) {
                 return null;
             }
+            JsonNode jsonNode = JacksonUtils.toObj(json);
```

Comment:
I think still should use file name to identify

Reply:
Okay